scripts/dev/generate_epJSON_schema/idd_parser.py

```diff
@@ -111,14 +112,17 @@ def parse_idd(data):
             eat_comment(data)
         elif token == TOKEN_GROUP:
             next_token(data)
-            eat_comment(data)
+            current_group_name = parse_line(data)
         else:
             obj_name = parse_string(data)
             if obj_name is None or obj_name == "":
                 return root
             obj_data = parse_obj(data)
             root['properties'][obj_name] = {}
             root['properties'][obj_name]['patternProperties'] = {}
+            if current_group_name is not None:
```

Comment:
Initializing to `None` and checking that here is great...but if you reset it to None in here, only the first object in the group will get this tag, right?

Reply:
You are correct. I forgot there was only one group declaration for multiple EP objects.